aicsimageio/aics_image.py

```diff
@@ -620,14 +620,14 @@ def __repr__(self) -> str:
         return f"<AICSImage [{type(self.reader).__name__}]>"
 
     @property
-    def cluster(self) -> Optional["distributed.LocalCluster"]:
+    def cluster(self) -> Optional["distributed.LocalCluster"]:  # noqa: F821
```

Comment:
Gah, having to ignore this is kind of frustrating 😕

Reply:
I think the new version of flake8 now checks for it because it wasn't happening before?? :shrug:

Reply:
Yea they just released a 3.8 compatible flake8 version that's got more stringent checks for newer stuff (hence the f-string warnings too)